python/ray/train/trainer.py

```diff
@@ -48,6 +50,12 @@ def noop():
     "torch": "TorchConfig"
 }
 
+BACKEND_ENV_VARS = {
```

Comment:
Can you add a comment here to document the purpose of this list?

Reply:
Also should we document these in the user docs somewhere?

Reply:
Done!